### DISCUSSION TITLE
Documentation about vselect and the cartridge?

### DISCUSSION THREAD

**Discussion by VKCsh** [sanni/cartreader#887] [Ideas & Suggestions]
Hi folks! 

On the [page about vselect](https://github.com/sanni/cartreader/wiki/Automatic-Voltage-Selection), I see the following text: 

> Note that you should not have a 3.3V cartridge in the reader when powering it on or resetting it, as it may briefly be sent 5V.

However, on the [overview page](https://github.com/sanni/cartreader/wiki/Overview), it says:

> You should also only insert or remove cartridges while the Cart Reader is turned off, and not have any cart inserted while you are doing a firmware update.

I'm not sure which of these is correct, they seem in conflict. I've read that the OSCR with vselect sends the lower voltage when a 5v system hasn't been selected, so I'm imagining it should be reasonably safe to plug a 3.3v game in before turning it on?

If this question was asked and answered already, I'm sorry. New to these forums and certainly could have missed it on my search. I very much enjoy my cartridge reader and I'm looking forward to using it more and more. Thank you!

**Comment by Ancyker** [comment]
Both are correct, however, you of course cannot follow both at the same time. The advice on the Vselect page is "more correct". 

The overview page says what it does because you might short a pin during cart insertion, but this is only a possibility and is not guaranteed; for most systems, this wouldn't hurt anything anyway because they were designed with kids in mind. The advice on the Vselect page is there because when power is first applied to the cart reader, Vselect will send 5V until the Arduino boots and tells it to send 3.3V. This always happens, it is guaranteed.

Edit: I should add that while the cart would be sent 5V, it would be very brief. It's not likely to cause issues, _but it could_. Both pages are attempting to err on the side of caution. Most if not all of the time between Vselect switching from 5V to 3.3V will be spent charging the capacitor which will reduce the amount of current available to the cartridge. I haven't measured it but I probably should since it's not that hard. When I made Vselect and wrote that guide I didn't have the tools to do so but I do now, so I probably will here soon.

**Reply by VKCsh** [reply]
I appreciate the response. So, if I'm understanding correctly and attempting to summarize: there are risks either way with 3.3V cartridges, but the risks are likely small.
